src/python/pants/backend/codegen/protobuf/python/grpc_python_plugin.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.core.util_rules.external_tool import ExternalTool
+from pants.engine.platform import Platform
+from pants.util.enums import match
+
+
+class GrpcPythonPlugin(ExternalTool):
+    """The grpc protobuf plugin."""
+
+    options_scope = "grpc_python_plugin"
```

Comment:
Is this only for Python, or this grpc works with every language? I suspect it'll work with other languages, given how `protoc` works with multiple languages out-of-the-box.

Reply:
It is specifically for python. The plugin is called `grpc_python_plugin`, there are others for other languages.

Reply:
Updated the docstring though, it wasn't expressing this correctly.